### PR TITLE
SDU-1854 create new interfaces for export meter readings instead of h…

### PR DESCRIPTION
**Pull request by eternalamenity**
…aving direction property, for backward compatibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -328,24 +330,23 @@ Name         | Value example | Description
 `delta_t`    |               | Elapsed time in seconds between the 'Meter Value' and the 'Previous Meter Value' measurements.
 `prv_data`   |               | Previous meter reading.
 `unit`       | "kWh"         | One of sup_units. For meter_unknown it will be a number.
-`direction`  | "export"      | Defines whether reading is based on consumption (import) or production (export). Applicable for every meter except for meter_elec.
 
 #### Important notes
 
-For backward compatibility, for service "meter_elec", interface "evt.meter.report" - reports should only support "import" direction.
+For backward compatibility, for service "meter_elec", interface "evt.meter.report" - reports should only support "import" direction. Created "evt.meter.export_report" to possibly report exported meter readings.
```
Comment (boguslaw-wojcik) [inline README.md:336]: Maybe a slightly improved version:

For backward compatibility service "meter_elec" reports imported values using "evt.meter.report" interface and exported values using "evt.meter.export_report" if applicable.

File: README.md [hunk 2]
```diff
@@ -313,13 +313,15 @@ Service name    | Units                                     | Description
...
-out  | evt.meter.report              | float      | unit, prv_data, delta_t, direction |
-out  | evt.meter_ext.report          | float_map  |                                    | [Extended meter report](#extended-report-object) with up to 17 data points
-in   | cmd.meter_ext.get_report      | null       |                                    | Request extended report
+Type | Interface                     | Value type | Properties              | Description
+-----|-------------------------------|------------|-------------------------|-------------
+in   | cmd.meter.get_report          | string     |                         | Value - is a unit. May not be supported by all meters.
+in   | cmd.meter.get_export_report   | string     |                         | Value - is an export unit. May not be supported by all meters.
+in   | cmd.meter.reset               | null       |                         | Resets all historical readings.
+out  | evt.meter.report              | float      | unit, prv_data, delta_t | Returns consumption meter report.
+out  | evt.meter.export_report       | float      | unit, prv_data, delta_t | Returns production meter report.
```
Comment (boguslaw-wojcik) [inline README.md:322]: Maybe unify usage of `import/export` and `consumption/production`, use either one pair throughout entire service or use `import/consumption` and `export/production` together to give the best explanatory value.

**Review by boguslaw-wojcik — DISMISSED** [review]
Suggested possible improvements.

File: README.md (hunk 1 above)
Comment (eternalamenity) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (eternalamenity) [reply README.md]: using pair together

**Review by boguslaw-wojcik — APPROVED** [review]

**Pull request merged**